webserver/webserver/src/main/java/io/helidon/webserver/ClassPathContentHandler.java

```diff
@@ -92,7 +92,9 @@ boolean doHandle(Http.RequestMethod method, String requestedPath, ServerRequest
         LOGGER.finest(() -> "Requested class path resource: " + resource);
 
         // this MUST be done, so we do not escape the bounds of configured directory
-        String requestedResource = URI.create(resource).normalize().toString();
+        // We use multi-arg constructor so it performs url encoding
+        URI myuri = new URI(null, null, resource, null);
```

Comment:
You don't need the intermediate variable right ? It can just be `new URI(null, null, resource, null).normalize().getPath()`

Reply:
I find this way more readable  with multi-arg constructor. Easier to step through with a debugger. If exception is thrown you know the method it came from. Compiler will optimize it for me.